ProphetBot/models/embeds/ref_embeds.py

```diff
@@ -1,6 +1,8 @@
+import io
```

Comment:
I don't see this new package used anywhere in this file. Is the import necessary?